pages/tasks/team_details.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+from pages.tasks.regions.task import Task
+
+
+class TeamDetailsPage(Base):
+
+    _name_locator = (By.CSS_SELECTOR, '.content-container > h2')
+    _task_list_header_locator = (By.CSS_SELECTOR, 'div.task-list-container > h3')
+    _available_tasks_list_locator = (By.CSS_SELECTOR, '.task-list > li')
+
+    @property
+    def _page_title(self):
+        return '%s | Mozilla One and Done' % self.name
+
+    @property
+    def name(self):
+        # need to remove ' Team' from the element
+        return self.selenium.find_element(*self._name_locator).text[:-5]
```

Comment:
This seems particularly fragile to me. Perhaps it would be better to have a property for `page_heading` and then in the test check that the `page_heading` contains the expected team name, rather than trying to allow the page object to return the team name when the markup doesn't really support that. This is another example where changing the page markup (e.g., by adding a `<span>` around the team name) could make it easier to test, if that's something you're interested in doing.